### PR TITLE
Buffs HoS jacket cold resistance

### PR DESCRIPTION
**Pull request by ryebek**
## About the PR 
Gives the HoS jacket 35% cold resistance, same as a security winter coat. Also gives it a nice new examine text, so the HoS can be bullied for their cosy and warm jacket.

## Why's this needed? 
The HoS jacket currently sacrifices a ton of armor for style points, while this is cool and all, it doesn't make sense that the jacket has 5% cold resistance, less than the HoS hat. This means theres more reason to pick the coat than just "style points".

### CODE REVIEW

**Comment by Kyle2143** [comment]
I like it, but isn't the blanket usually how people refer to the HoS cape?

**Comment by ryebek** [comment]
You're right, I'll drop the blanket bit, can't be stealing the HoS capes thunder.

File: code/obj/item/clothing/suits.dm [hunk 1]
```diff
@@ -339,14 +339,15 @@
 
 /obj/item/clothing/suit/det_suit/hos
 	name = "HoS jacket"
-	desc = "A slightly armored jacket favored by security personnel."
+	desc = "A slightly armored jacket favored by security personnel. It looks cosy and warm,you could probably sleep in this if you wanted to!"
```
Comment (ZeWaka) [inline code/obj/item/clothing/suits.dm:342]:
```suggestion
	desc = "A slightly armored jacket favored by security personnel. It looks cozy and warm; you could probably sleep in this if you wanted to!"
```

Comment (Kyle2143) [reply code/obj/item/clothing/suits.dm:342]: Rekt

**Pull request merged**